<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>801</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 85, 127)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;Olan's Recipe Application (21318204)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>530</y>
      <width>295</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="nutritionButtons">
     <item>
      <widget class="QRadioButton" name="hideNutrition">
       <property name="text">
        <string>Don't Show Nutrition</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="showNutrition">
       <property name="text">
        <string>Show Nutrition</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="recipeLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 0)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;HELLO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>411</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="ingredientsLayout"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>631</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="mealLayout"/>
   </widget>
   <widget class="QLabel" name="mealOption">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;SELECT A MEAL OPTION&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>231</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="carbsValueLabel">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 255, 255)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fatValueLabel">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 255, 255)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="proteinValueLabel">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 255, 255)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="caloriesValueLabel">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 255, 255)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSlider" name="timeSlider">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>179</y>
      <width>20</width>
      <height>371</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SET TIMER WITH VERTICAL SLIDER</string>
    </property>
   </widget>
   <widget class="QLabel" name="recipeCount">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>489</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>169</width>
      <height>129</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="verticalLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="highCal">
       <property name="text">
        <string>HighCal</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="lowCal">
       <property name="text">
        <string>LowCal</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHome">
    <property name="title">
     <string>Home</string>
    </property>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionRepo_Link"/>
   </widget>
   <addaction name="menuHome"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionexit">
   <property name="icon">
    <iconset>
     <normaloff>exit.png</normaloff>exit.png</iconset>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionRepo_Link">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>GitHub_logo.png</normaloff>GitHub_logo.png</iconset>
   </property>
   <property name="text">
    <string>Repo Link</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
